lib/twilio/device.ts

```diff
@@ -1556,8 +1556,11 @@ class Device extends EventEmitter {
     this._callSinkIds = sinkIds;
     const call = this._activeCall;
     return call
-      ? call._setSinkIds(sinkIds)
-      : Promise.resolve();
+        ? call._setSinkIds(sinkIds)
+        : new Promise(resolve => {
+          this._calls.map(_call => _call._setSinkIds(sinkIds));
+          resolve();
+        });
```

Comment:
The unit tests that Twilio has passed for this. Could be missing a test that checks that no other call beside master is updated.
Maybe that tests was never necessary.